Libraries/Image/RCTImageUtils.m

```diff
@@ -314,7 +314,7 @@ BOOL RCTUpscalingRequired(CGSize sourceSize, CGFloat sourceScale,
   return (__bridge_transfer id)imageProperties;
 }
 
-NSData *__nullable RCTGetImageData(CGImageRef image, float quality)
+NSData *__nullable RCTGetImageData(CGImageRef image, float quality, NSInteger orientation)
 {
   NSDictionary *properties;
```

Comment:
This is for `CGImageDestinationAddImage` and `properties` is the correct name.
https://developer.apple.com/documentation/imageio/1464962-cgimagedestinationaddimage